src/OrchardCore.Modules/OrchardCore.Contents/Deployment/ExportContentToDeploymentTarget/ExportContentToDeploymentTargetAdminMenu.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Localization;
+using OrchardCore.Navigation;
+
+namespace OrchardCore.Contents.Deployment.ExportContentToDeploymentTarget
+{
+    public class ExportContentToDeploymentTargetAdminMenu : INavigationProvider
+    {
+        private readonly IStringLocalizer S;
+
+        public ExportContentToDeploymentTargetAdminMenu(IStringLocalizer<AdminMenu> localizer)
+        {
+            S = localizer;
+        }
+
+        public Task BuildNavigationAsync(string name, NavigationBuilder builder)
+        {
+            if (!String.Equals(name, "admin", StringComparison.OrdinalIgnoreCase))
+            {
+                return Task.CompletedTask;
+            }
+
+            builder
+                .Add(S["Configuration"], configuration => configuration
+                    .Add(S["Import/Export"], S["Import/Export"], import => import
+                        .Add(S["Settings"], settings => settings
+                            .Add(S["Export Content To Deployment Target"], "am-" + S["Export Content To Deployment Target"], deployment => deployment
```

Comment:
Please merge dev and use .PrefixPosition() instead of `"am-" +`.

Reply:
Done